### PR TITLE
feat(dws): add new data source to query volumes of the logical cluster

### PR DESCRIPTION
**Pull request by wuzhuanhong**
**What this PR does / why we need it**:

Add new data source to query volumes corresponding to logical cluster.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. add new data source.
2. add corresponding to document and acceptance test.
```

## PR Checklist

* [x] Tests added/passed.

```
make testacc TEST='./huaweicloud/services/acceptance/dws' TESTARGS='-run TestAccDataSourceLogicalClusterVolumes'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dws -v -run TestAccDataSourceLogicalClusterVolumes -timeout 360m -parallel 4
=== RUN   TestAccDataSourceLogicalClusterVolumes_basic
=== PAUSE TestAccDataSourceLogicalClusterVolumes_basic
=== CONT  TestAccDataSourceLogicalClusterVolumes_basic
--- PASS: TestAccDataSourceLogicalClusterVolumes_basic (27.66s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dws       27.717s
```

* [x] Documentation updated.
* [x] Schema updated.
* [ ] CheckDeleted.

  - **a. During query operation (Read Context)**
    aa. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

    
  - **b. During delete/disassociate/unbind operation (Delete Context)**
    ba. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

### CODE REVIEW

**Comment by wuzhuanhong** [comment]
make testacc TEST='./huaweicloud/services/acceptance/dws' TESTARGS='-run TestAccDataSourceLogicalClusterVolumes_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dws -v -run TestAccDataSourceLogicalClusterVolumes_basic -timeout 360m -parallel 4
=== RUN   TestAccDataSourceLogicalClusterVolumes_basic
=== PAUSE TestAccDataSourceLogicalClusterVolumes_basic
=== CONT  TestAccDataSourceLogicalClusterVolumes_basic
--- PASS: TestAccDataSourceLogicalClusterVolumes_basic (32.64s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dws       32.741s

File: docs/data-sources/dws_logical_cluster_volumes.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The data source ID.
+
+* `volumes` - The list of the disk volumes corresponding to the logical cluster.
+
+  The [volumes](#volumes_struct) structure is documented below.
```
Comment (Lance52259) [inline docs/data-sources/dws_logical_cluster_volumes.md]:
```suggestion
  The [volumes](#logical_cluster_volumes_struct) structure is documented below.
```

Comment (wuzhuanhong) [reply docs/data-sources/dws_logical_cluster_volumes.md]: fixed

**Pull request merged**